src/mgr/ActivePyModules.cc

```diff
@@ -1080,3 +1082,14 @@ void ActivePyModules::cluster_log(const std::string &channel, clog_type prio,
     clog->do_log(prio, message);
   }
 }
+
+void ActivePyModules::register_client(std::string_view name, uintptr_t radosp)
+{
+  std::lock_guard l(lock);
+
+  auto R = reinterpret_cast<librados::RadosClient*>(radosp);
```

Comment:
worth noting here that we are assuming this RadosClient never goes away?

Reply:
The RadosClient associated with the module persists as long as the mgr stays active. So this should be true.

Reply:
BTW, I figured out why the MDS wasn't blacklisting the clients. I had thought that the RADOS handle passed to libcephfs was used (messenger and all) when setting up the `Client`. However, only the `cct` was used. So we have yet another messenger instance created. I'm working on a fix.